### DISCUSSION TITLE
Does not work with Cura?

### DISCUSSION THREAD

**Discussion by gitbreiti** [nlef/moonraker-telegram-bot#192] [General]
When I use SuperSlicer the status in Telegram works fine. But when I use Cura, notifications no longer come. 

Can anyone help me?

**Comment by aka13-404** [comment]
Would you kindly open an issue and attach the g-code where notifications don't work for you?

**Reply by gitbreiti** [reply]
After I installed "Cura2MoonrakerPlugin" it works.
